Some small fixes

lukedirtwalker opened the pull request:
-One is compilation problem with clang
-The other is an adaption of the boost python library flag, to be more system independent.
<a href='#crh-start'></a><a href='#crh-data-%7B%7D'></a>
<a href='https://www.codereviewhub.com/'><img src='http://www.codereviewhub.com/site/github-bar.png' height=40></a>
<a href='https://www.codereviewhub.com/dimitar-asenov/Envision/pull/125?approve=1'><img src='http://www.codereviewhub.com/site/github-approve.png' height=26></a>&nbsp;<a href='https://github.com/dimitar-asenov/Envision/pull/125'><img src='http://www.codereviewhub.com/site/github-refresh.png' height=26></a>
<a href='#crh-end'></a>

lukedirtwalker committed:
Fix string concatenation with int

lukedirtwalker committed:
Find the correct libboost_python lib on linux systems.

dimitar-asenov commented:
This pull request is now managed with CodeReviewHub. <a href='https://github.com/dimitar-asenov/Envision/pull/125'>Refresh</a>.<a href='#crh-update-none'></a>

lukedirtwalker committed:
Add windows and mac build flags in InformationScripting.pro

Additionally use python3-config instead of the detailed version (e.g.
python3.4-config) this means less adaption work between python versions.
